src/components/InfiniteScrollPaginator/InfiniteScroll.tsx

```diff
@@ -130,14 +123,17 @@ export const InfiniteScroll = (props: PropsWithChildren<InfiniteScrollProps>) =>
     const scrollElement = scrollComponent.current?.parentNode;
     if (!scrollElement) return;
 
+    const scrollListener = () => scrollListenerRef.current?.();
+
     scrollElement.addEventListener('scroll', scrollListener, useCapture);
     scrollElement.addEventListener('resize', scrollListener, useCapture);
+    scrollListener();
```

Comment:
Note that we'll only load a single additional page of messages in case the first page is not enough to cause overflow of the message list.

To keep loading pages until there is overflow, or there is no more pages to load, we can do something like this:

```ts
  useLayoutEffect(() => {
    if (!isLoading) {
      scrollListenerRef.current?.();
    }
  }, [isLoading]);
```

However, I don't see a way to have the same behavior in the `VirtualizedMessageList`, and since our default page size is rather big (100 message), I think it's better to keep things consistent.